pages/[id]/delegate.tsx

```diff
@@ -64,62 +67,70 @@ const Delegate: NextPage = () => {
       <Head>
         <title>Cross Chain Voting: Delegate</title>
       </Head>
-      <div className="flex justify-center align-center self-center h-full w-1/3">
+      <div className="flex justify-center align-center self-center h-full max-w-lg">
         <div className="flex flex-col m-4 w-full gap-5">
-          <div className="flex flex-col self-center">
-            <h1 className="py-5 block text-xl">Delegate on {mounted && config.l2.chain.name}</h1>
-            <div className="flex w-full py-4 justify-between">
-              <div className="flex flex-col">
-                <div>Token Balance</div>
-                <div className="flex align-center p-4 gap-1">
-                  <Image
-                    height="32"
-                    width="32"
-                    src={config.l2.tokenLogo}
-                    alt={`${config.name}'s Governor token logo`}
-                  />
-                  <div className="self-center">{mounted && (l2.token?.formatted || 0)}</div>
-                  <div className="self-center">{mounted && (l2.token?.symbol || '---')}</div>
+          <CardWithHeader header={`Delegate on ${mounted && config.l2.chain.name}`} className="w-full">
+            <div className="flex items-center gap-1.5 text-gray-600 py-2 mb-2 ">
+              <InformationCircleIcon className="h-4" />
+              <p className="text-sm ">
+                {mounted && l2.token?.symbol} determines {config.name} voting power on{' '}
```

Comment:
<img width="537" alt="Screenshot 2023-11-16 at 11 51 31 AM" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/5423313/fe73247e-ec1e-4a53-a8e7-53960c916677">

> [ blank ] determines PoolTogether voting power on Optimism Goerli

currently, token info is gotten via `useBalances` hook, but since we don't have an address, we don't have a balance, therefore this token symbol isn't loading

Reply:
tracking in #140